py2hwsw/lib/hardware/arith_logic/iob_edge_detect/iob_edge_detect.py

```diff
@@ -5,22 +5,34 @@
 
 def setup(py_params_dict):
     attributes_dict = {
-        "generate_hw": False,
+        "generate_hw": True,
+        "confs": [
+            {
+                "name": "EDGE_TYPE",
+                "descr": "Edge detection type",
```

Comment:
```suggestion
                "descr": "Edge detection type. Options: "rising", "falling", "both"",
```
We should keep the options in the description for easier usage.

Reply:
update to:
```python
"descr": "Edge detection type. Options: 'rising', 'falling', 'both'",
```
for proper string escaping.
Can't use `"` in the middle of a string delimited by `"<string>"` otherwise it gets closed before in python